sdk/test-utils/recorder/src/utils/index.ts

```diff
@@ -474,12 +474,11 @@ export function maskAccessTokenInNockFixture(fixture: string): string {
   }
   // Replaces only if the content-type is json
   if (isContentTypeInNockFixture(fixture, jsonContentTypes)) {
```

Comment:
What about browser recordings?

Reply:
Currently the ACR tests don't run on browsers yet so I cannot test the scenario. Will come back to it later.